.gitignore

```diff
@@ -56,6 +56,13 @@ MetricsData
 .temp*
 #
 # ----------------------------------------------------------------------
+# ML Files
+# ----------------------------------------------------------------------
+ml/*.out
+ml/*.model
+venv/*
+#
+# ----------------------------------------------------------------------
```

Comment:
Why does this PR include such unrelated changes?

Reply:
It was because it had been confusing git. I'll revert it if you're uncomfortable with it